### PR TITLE
Reorganize packages

### PR DESCRIPTION
**Pull request by jasonburkert-ccpo**
Reorganizes packages for increased clarity.
The catalyst for this was a discussion about where files for route `/taskOrderFiles` will live in support of ticket AT-6417.

Per team discussion on 8/12:
TARGET STRUCTURE TODAY
```
./infrastructure/
./infrastructure/package.json

./packages/api/
./packages/api/package.json
./packages/api/portfolioDrafts/
./packages/api/taskOrderFiles/
./packages/api/utils/
./packages/api/models/

./packages/filescanner/
./packages/filescanner/package.json
```

FUTURE MODIFICATIONS - Move `models` to its own shareable package
```
./packages/models/
./packages/models/package.json
```

Replaces #64.

### CODE REVIEW

**Comment by jasonburkert-ccpo** [comment]
Deployed stack and tested all existing functions. Verified behavior is as expected.

**Review by jeffsegal-ccpo — APPROVED** [review]
Looks great!

**Comment by jasonburkert-ccpo** [comment]
@kylelaker-ccpo Jeff wants to give you a chance to review and merge this.

File: lerna.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "packages": [
     "infrastructure",
-    "applications/**"
+    "packages/api"
```
Comment (kylelaker-ccpo) [inline lerna.json:4]:
```suggestion
    "packages/**"
```

Is there a specific reason we'd want to enumerate every package? l am concerned it'd be an easy thing to forget in the future

Comment (jasonburkert-ccpo) [reply lerna.json:4]: No.  I agree could be easy to forget.  But I found plenty of other places where we'd have to specify resources in a package.

**Review by kylelaker-ccpo — APPROVED** [review]
Thank you!

**Pull request merged**